unity-sdk/AndroidPlugin/OptimoveUnityPlugin/src/main/java/com/optimove/unity/plugin/UnityProxy.java

```diff
@@ -1,4 +1,376 @@
 package com.optimove.unity.plugin;
 
+import android.content.Context;
+
+import com.optimove.android.Optimove;
+import com.optimove.android.optimobile.InAppDeepLinkHandlerInterface;
+import com.optimove.android.optimobile.InAppInboxItem;
+import com.optimove.android.optimobile.InAppInboxSummary;
+import com.optimove.android.optimobile.OptimoveInApp;
+import com.unity3d.player.UnityPlayer;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.net.URL;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.TimeZone;
+
 public class UnityProxy {
+    private static JSONObject pendingPush;
+
+    public static void setUserId(String userId) {
+        try {
+            Optimove.getInstance().setUserId(userId);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void setUserEmail(String userEmail) {
+        try {
+            Optimove.getInstance().setUserEmail(userEmail);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void reportEvent(String eventName) {
+        try {
+            Optimove.getInstance().reportEvent(eventName);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void reportEvent(String eventName, String eventParamsString) {
```

Comment:
eventParamsString -- no need to include type into name, it already has `String` type

Reply:
renamed.